man/rgbasm.5

```diff
@@ -30,13 +30,13 @@ but any program that processes RGBDS object files (described in
 .Xr rgbds 5 )
 can be used in its place.
 .Sh SYNTAX
-The syntax is line-based, just as in any other assembler, meaning that you do one instruction or directive per line:
+The syntax is line-based, meaning that you put one instruction or directive per line:
```

Comment:
Well, the entire point is that this isn't true anymore. I wanted to suggest a different wording, but the meaning of "line-based" is unclear in the first place, so I feel this introduction needs to be rewritten entirely.

Reply:
Revised:
![image](https://github.com/gbdev/rgbds/assets/35663410/41fadac5-7f02-443e-9bee-c3519902e852)